### ISSUE TITLE
Multilevel logistic regression

### ISSUE THREAD

**Issue by tom-christie** [bayespy/bayespy#53]
Are there any examples of using BayesPy to do multilevel logistic regression? This seems like it'd be straightforward to do if you could define a variable as an arbitrary function of another, like 

    p[i] <- 1 / (1 + exp(-z[i]))

but I don't know whether that's possible.

**Comment by jluttine** [comment]
Nope, it's not currently possible to use arbitrary functions. BayesPy is currently limited to the conjugate exponential family. I have plans to add support for non-conjugate nodes but we'll see when that happens.

I added a separate general issue for that: https://github.com/bayespy/bayespy/issues/54 in case you want to follow the progress by subscribing to the issue. But it will probably take some time until I find the time to implement it. So I guess my suggestion would be to take a look on Stan.
